samples/EmbeddedViewSample.Web/Startup.cs

```diff
@@ -19,20 +19,22 @@ public void ConfigureServices(IServiceCollection services)
 
             services.Configure<RazorViewEngineOptions>(options =>
             {
-                options.FileProvider = new EmbeddedFileProvider(GetType().GetTypeInfo().Assembly, "RazorEmbeddedViewsWebSite.EmbeddedResources");
+                // Base namespace matches the resources added to the assembly from the EmbeddedResources folder.
+                options.FileProvider = new EmbeddedFileProvider(
+                    GetType().GetTypeInfo().Assembly,
+                    baseNamespace: "EmbeddedViewSample.Web.EmbeddedResources");
             });
         }
 
         public void Configure(IApplicationBuilder app)
         {
-            app.UseCultureReplacer();
-
             app.UseMvc(routes =>
             {
                 routes.MapRoute("areaRoute", "{area:exists}/{controller}/{action}");
-                routes.MapRoute("default",
-                                "{controller}/{action}/{id?}",
-                                new { controller = "Home", action = "Index" });
+                routes.MapRoute(
+                    "default",
+                    "{controller}/{action}/{id?}",
+                    new { controller = "Home", action = "Index" });
```

Comment:
With the controller renames, `@Url.Action()` generates just `"/"` in `Index.cshtml`. @rynowak what is your opinion on the slightly-ambiguous expected results (`"Hello test-user, this is /"`) in the associated tests? See https://github.com/aspnet/Mvc/pull/3713/files#diff-2f8f009c117b95d88d3ea7fe56e0032eR23 for example.